mythril/laser/ethereum/state/machine_state.py

```diff
@@ -123,7 +125,10 @@ def calculate_extension_size(self, start: int, size: int) -> int:
         """
         if self.memory_size > start + size:
             return 0
-        return start + size - self.memory_size
+        new_size = utils.ceil32(start + size) // 32
```

Comment:
Can you add a comment saying why you do this trick with division and ceil.

I'm assuming it's because of word alignment